features/channels/channel_theme_selection.feature

```diff
@@ -0,0 +1,29 @@
+@ui-themes
+Feature: Using themes
+    In order to allow customizing channels' appearance
+    As a store owner
+    I want to be able to set theme per channel
+
+    Background:
+        Given the store is operating on a single "France" channel
+        And there are "Maverick Meerkat" and "Vivid Vervet" themes defined
+
+    Scenario: None of the themes are used by default
+        When I create a new channel "Poland"
+        Then that channel should not use any theme
+
+    Scenario: Setting a theme on a channel
+        When I set "France" channel theme to "Maverick Meerkat"
+        Then that theme should be used
+
+    Scenario: Unsetting a channel theme
+        Given "France" channel is using "Maverick Meerkat" theme
+        When I unset theme on that channel
+        Then that channel should not use any theme
+
+    Scenario: Every channel should use different theme
+        Given there is another channel "Poland"
+        When I set "Maverick Meerkat" theme to be used by "France" channel
+        And I set "Vivid Vervet" theme to be used by "Poland" channel
+        Then "France" channel should use "Maverick Meerkat" theme
+        And "Poland" channel should use "Vivid Vervet" theme
```

Comment:
I think this scenario should be in separate feature, it is not about the management itself. Actor should be Visitor for this scenario too. Wdyt?